src/ts/components/ArcComposites.tsx

```diff
@@ -117,7 +117,7 @@ export const WebBase = (props: WebBaseProps) => (
                             }
                         });
                         resolve({ map, view });
-                    }).otherwise((err) => {
+                    }).catch((err) => {

```

Comment:
is this a 4.15 thing? If so is the plan to force everyone to 4.15+ starting w/ the next release of react-arcgis?

Reply:
oops, just read the description